src/types/queries.rs

```diff
@@ -284,6 +284,16 @@ pub enum PendingUpdateEffect {
     CooldownParameters(CooldownParameters),
     #[serde(rename = "timeParametersV1")]
     TimeParameters(TimeParameters),
+    #[serde(rename = "gasRewardsCPV2")]
+    GasRewardsCPV2(GASRewardsCPV2),
+    #[serde(rename = "TimeoutParametersCPV2")]
+    TimeoutParametersCPV2(TimeoutParameters),
+    #[serde(rename = "minBlockTimeCPV2")]
+    MinBlockTimeCPV2(Duration),
+    #[serde(rename = "blockEnergyLimitCPV2")]
+    BlockEnergyLimitCPV2(Energy),
+    #[serde(rename = "finalizationCommitteeParametersCPV2")]
+    FinalizationCommitteeParametersCPV2(FinalizationCommitteeParameters),
```

Comment:
I am not sure where these JSON names are coming from. They are not really returned in the node, and also they don't fit the naming for the existing ones. The existing ones don't have CPV.

Reply:
Right, they are not coming from anywhere, since this serde implementation is mainly needed by the old json based gRPC API.
I'll remove the suffix